package.json

```diff
@@ -261,15 +261,16 @@
     "escape-string-regexp": "^5.0.0",
     "file-saver": "^2.0.5",
     "firebase": "^8.10.1",
+    "immer": "^10.1.3",
```

Comment:
This is fine so far, but if the authoring specific dependencies start to get large we should figure out how to separate them out.